.github/workflows/release.yml

```diff
@@ -0,0 +1,100 @@
+name: trino-python-client release
+
+on:
+  workflow_dispatch:
+
+env:
+  PYTHON_VERSION: "3.9.5"
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{env.PYTHON_VERSION}}
+
+      - name: Test release
+        run: |
+          python3 -m venv .venv
+          . .venv/bin/activate
+          pip install .
+          pip install twine wheel setuptools
+          ./setup.py sdist bdist_wheel
+          pip install dist/trino-*.tar.gz
+          pip install dist/trino-*-py3-none-any.whl
+          twine check dist/trino-*-py3-none-any.whl dist/trino-*.tar.gz
```

Comment:
This is a very surface level test and doesn't verify if the package works.

We should consider some form of "product-tests" here too (a follow up is fine).

Reply:
We could run all tests here by executing `tox` instead of doing package checks (or run tests and then do package checks).

Reply:
> I believe that PyPi does allow staging releases - we should use it for the test job and then install from there and verify if the package works.

That would be some form of `product-test` which you mentioned. 

Proposed flow for `test` job:
1. Package checks
2. Stage release to PyPi
3. Install from PyPi
4. Run all tests by running `tox`

Reply:
> I believe that PyPi does allow staging releases -

you mean https://test.pypi.org/ ?